project/src/components/offer-card/offer-card.tsx

```diff
@@ -22,6 +22,10 @@ function OfferCard({ className, info, onMouseOver }: OfferCardProps) {
   const classNameInfo = `${className}__card-info place-card__info`;
   const offerRoute = AppRoute.Room.replace(':id', id);
 
+  const listItemHoverHandler = () => {
```

Comment:
```suggestion
  const handleListItemHover = () => {
```
`handle` для локальных коллбэков
`on` для коллбэков через пропсы

Reply:
исправила